### DISCUSSION TITLE
List Page Deletions by Last Actioned Time

### DISCUSSION THREAD

**Discussion by falo2k** [gotson/komga#1609] [Ideas]
Appreciate there may be a bit of work here if it's not already logged, but I think it would be handy if we could list the "Known" section of the Duplicate Page management by the time a deletion was last performed.  Once the section gets particularly large, it's almost impossible to find an entry that might be causing problems (e.g. something accidentally added).  To allow resolving such issues, being able to see what's been most recently found and deleted would at least let a user re-process a raw file and then find the entry they may want to change.

**Comment by gotson** [comment]
why would you want to do that ? i don't understand the use case

**Reply by falo2k** [reply]
To try to hunt down additions to the deletion list that you want to remove (e.g. if they've been added by mistake).  Once they've been added, it's very difficult to go back and find a specific entry.  Stats on the impact the page removals have had (matches, size, etc.) are great, but not particularly useful for finding specific entries.  It could probably do with being able to sort on when the page filter was set up as well, to allow a simple mechanism for correcting recent mistakes.

**Reply by gotson** [reply]
You can already see the deleted pages in the History section

**Reply by falo2k** [reply]
But from there you have no ability to link back to the item causing the deletions that I can see.

**Reply by gotson** [reply]
yes true, you can see the page deleted but cannot action on it

**Reply by gotson** [reply]
i will add 2 sort criterias for _date added_ and _date modified_. When a page is deleted, the count is increased, and thus the modified date will be updated. But it would also be updated when the action is changed.

between this and the History tab, you should have enough information to find the known page hash i guess

**Reply by falo2k** [reply]
> i will add 2 sort criterias for _date added_ and _date modified_. When a page is deleted, the count is increased, and thus the modified date will be updated. But it would also be updated when the action is changed.
> 
> between this and the History tab, you should have enough information to find the known page hash i guess

Sounds awesome, ta very much!
